lib/home/view/home_view.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:qr_code/home/widgets/widgets.dart';
+
+class HomeView extends StatelessWidget {
+  const HomeView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context).home),
```

Comment:
**note**: this is impressive 😄 

One thing that we usually do is also to create an extension method like this

```dart
extension LocalizationExt on BuildContext {
  AppLocalizations get l10n => AppLocalizations.of(this);
}
```

But yeah just to say, all good here 🙂

Reply:
Wow that extension is really amazing 👏🏽